routes/Signup.py

```diff
@@ -21,4 +21,10 @@ def getResponse(self, request: Request) -> bytes:
             if form["file-type"] != "png" and form["file-type"] != "jpg" and form["file-type"] != "jpeg":
                 return Route.buildResponse(400,{"Content-Type":"text/plain"},b'Submitted file must be an image')
 
-            return Route.buildResponse(200,{"Content-Type":"text/plain"},b'Account created')
\ No newline at end of file
+            return Route.buildResponse(200,{"Content-Type":"text/plain"},b'Account created')
+        elif request.req_type == "GET":
+            f = open("./public/signup.html", "rb")
+            file = f.read()
+            file_length = str(len(file))
+
+            return Route.buildResponse(200,{"Content-Type":"text/html","X-Content-Type-Options":"nosniff","Content-Length":file_length},file)
```

Comment:
same thing here,
* Encase in try/except
* In case of except, return 500 response
* Get rid of content-type-options and content-length